test/rspack-dev-tests-manifest.json

```diff
@@ -2509,11 +2509,12 @@
   },
   "test/development/app-dir/missing-required-html-tags/index.test.ts": {
     "passed": [
-      "app-dir - missing required html tags should display correct error count in dev indicator",
+      "app-dir - missing required html tags should display correct error count in dev indicator"
+    ],
+    "failed": [
       "app-dir - missing required html tags should hmr when you fix the error",
       "app-dir - missing required html tags should show error overlay"
```

Comment:
There is unfortunately no way to tell if this is flakey or a proper regression @bgw @wbinnssmith